tutorials/spanner-oc/java/src/main/java/com/example/spanner/App.java

```diff
@@ -0,0 +1,194 @@
+        for (int i = 0; i < 3; i++) {
+          String up = i + "-" + (System.currentTimeMillis() / 1000) + ".";
+          List<Mutation> mutations = Arrays.asList(
+            playerMutation("Poke", "Mon", up + "poke.mon@example.org",
+              "f1578551-eb4b-4ecd-aee2-9f97c37e164e"),
+            playerMutation("Go", "Census", up + "go.census@census.io",
+              "540868a2-a1d8-456b-a995-b324e4e7957a"),
+            playerMutation("Quick", "Sort", up + "q.sort@gmail.com",
+              "2b7e0098-a5cc-4f32-aabd-b978fc6b9710")
+          );
+
+          // write to Spanner
+          dbClient.write(mutations);
+        }
+        // [END spanner_insert_data]
+      }
+    } catch (Exception e) {
+      System.out.print("Exception while adding player: " + e);
+    } finally {
+      // Closes the client which will free up the resources used
+      spanner.close();
+    }
+  }
+
+  // [START spanner_create_database]
+  private static void createDatabase(DatabaseAdminClient dbAdminClient, DatabaseId id) {
+    OperationFuture<Database, CreateDatabaseMetadata> op =
+      dbAdminClient.createDatabase(
+        id.getInstanceId().getInstance(),
+        id.getDatabase(),
+        Collections.singletonList(
+          "CREATE TABLE Players (\n"
+            + "  first_name STRING(1024),\n"
+            + "  last_name  STRING(1024),\n"
+            + "  email   STRING(1024),\n"
+            + "  uuid STRING(1024)\n"
+            + ") PRIMARY KEY (email)"));
+    try {
+      // Initiate the request which returns an OperationFuture.
+      Database db = op.get();
+      System.out.println("Created database [" + db.getId() + "]");
+    } catch (ExecutionException e) {
+      // If the operation failed during execution, expose the cause.
+      throw (SpannerException) e.getCause();
+    } catch (InterruptedException e) {
+      // Throw when a thread is waiting, sleeping, or otherwise occupied,
+      // and the thread is interrupted, either before or during the activity.
+      throw SpannerExceptionFactory.propagateInterrupt(e);
+    }
+  }
+  // [END spanner_create_database]
+
+  // [START spanner_insert_data]
+  private static Mutation playerMutation(String firstName, String lastName, String email,
+    String uuid) {
+    return Mutation.newInsertBuilder("Players")
+      .set("first_name")
+      .to(firstName)
+      .set("last_name")
+      .to(lastName)
+      .set("uuid")
+      .to(uuid)
+      .set("email")
+      .to(email)
+      .build();
+  }
+  // [END spanner_insert_data]
+
+  private static void sleep(int ms) {
+    try {
+      Thread.sleep(ms);
+    } catch (Exception ignored) { }
+  }
+
+  public static void main(String ...args) throws Exception {
+    configureOpenCensusExporters();
+    doSpannerOperations();
+
+    // IMPORTANT: do NOT exit right away. Wait for a duration longer than reporting
+    // duration (5s) to ensure spans are exported. Spans are exported every 5 seconds.
+    sleep(5100);
```

Comment:
Metrics are exported every 60s by default. You may want to update https://github.com/GoogleCloudPlatform/community/pull/887/files#diff-2c38eb47f2a5d48471faa52a05eac183R76.